### DISCUSSION TITLE
Missing Windows.Imaging namespace

### DISCUSSION THREAD

**Discussion by jennykim515** [microsoft/psi#174] [Q&A] [answered]
Hi! I am getting familiarized with psi using the provided samples. I'm now on the webcam with the audio sample, and I'm unable to include the System.Windows.Media.Imaging namespace and I'm using NET Core 3.1. Does anyone know the solution to this?

**Comment by sandrist** [comment, marked as answer]
I think the easiest approach will be to recreate the project as a WPF app targeting .NET Framework 4.7.2. That should allow you to include System.Windows.Media.Imaging. The .csproj for the sample should look something like the following at the top:

```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>net472</TargetFramework>
    <OutputType>WinExe</OutputType>
    ...
```

**Reply by jennykim515** [reply]
Awesome thank you!
